qa/automation/CareerPortal.Tests.Scenarios/3. Ivanova Tests/3.1 CareerPortalTests_CVform.feature

```diff
@@ -0,0 +1,127 @@
+    And user presses the button "Send"
+    Then button "Attach CV" text should have color #ef413d
+    And CV form shoud not be sent
+
+Scenario: 3.1.6 User send CV form without checks checkbox "I agree to the personal data processing"
+    When user writes "Steve Jobs" in the Name field
+    And user writes "SteveJobs1@gmail.com" in the Email field
+    And user attaches CV file from path "cv_file_valid.pdf"
+    And user wtites "Hi, Quantori!!!" in the Message field
+    And user presses the button "Send"
+    Then checkbox label text should have color rgb(239, 65, 61)
+    And CV form shoud not be sent
+
+Scenario Outline: 3.1.7 User filled value in the Name field
+    When user writes <name> to the Name field
+    Then text below field Name should be <error text>
+    @DateSource:longNames.csv
+    Examples: 
+        |name                   |error text         |
+        |a a                    |                   |
+        |Ivan Ivanov            |                   |
+        |���� ������            |                   |
+        |Ivan Ivanov Ivanovich  |                   |
+        |Ivan Ivanov Ivanovich. |                   |
+        |Ivan-Ivanov Ivanovich  |                   |
+        |Ivan Ivanov Ivanovach2 |Invalid name format|
+        |Ivanov                 |Invalid name format|
+        |<Ivan Ivanov Ivanovach>|Invalid name format|
+
+Scenario Outline: 3.1.8 User filled value in the Email field
+    When user writes <email> to the Email field
+    Then text below field Email should be <error text>
+    @DateSource:longEmails.csv
+    Examples: 
+        |email             |error text          |
+        |Ivanov@mail.ru    |                    |
+        |Ivanov@gmail.com  |                    |
+        |Ivanov3@g3mail.com|                    |
+        |Ivanov            |Invalid email format|
+        |Ivanov@           |Invalid email format|
+        |Ivanova@mail.     |Invalid email format|
+        |@mail.ru          |Invalid email format|
+        |@.com             |Invalid email format|
+        |Iv<a>nov@gmail.com|Invalid email format|
+        |Ivanov@gmail.co3m |Invalid email format|
+
+Scenario Outline: 3.1.9 User filled value in the Message field
+    When user writes <message> to the Message field
+    Then text below field Message should be <error text>
+    @DateSource:longMessages.csv
+    Examples: 
+        |message     |error text|
+        |            |          |
+        |a           |          |
+        |<>!@#$%^&*()|          |
+        |Hello!      |          |
+        |������ ����!|          |
+
+Scenario Outline: 3.1.10 User attached valid CV file
+    When user attaches CV file from path <path to file>
+    Then text below button "Attach CV" should be <path to file>
+
+    Examples: 
+        |path to file          |
+        |cv_file_valid.pdf     |
+        |cv_file_valid.doc     |
+        |cv_file_valid.docx    |
+        |cv_file_valid.rtf     |
+        |cv_file_valid_10mb.pdf|
+    
```

Comment:
I decided to divide Scenarios to checking valid and invalid files, because in the first case we will analyze text below attach button, in second case - text inside opened dialog. Next Scenario is about checking invalide files.